{
	"id": "CVE-2023-1981",
	"vulnerabilities": [
		{
			"id": "CVE-2023-1981",
			"description": "A vulnerability was found in the avahi library. This flaw allows an unprivileged user to make a dbus call, causing the avahi daemon to crash.",
			"severity": [
				{
					"type": "cvss_v31",
					"source": "security.alpinelinux.org",
					"cvss_v31": {
						"vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H",
						"base_score": 5.5,
						"base_severity": "MEDIUM",
						"temporal_score": 5.5,
						"temporal_severity": "MEDIUM",
						"environmental_score": 5.5,
						"environmental_severity": "MEDIUM"
					}
				}
			],
			"references": [
				{
					"source": "security.alpinelinux.org",
					"url": "https://security.alpinelinux.org/vuln/CVE-2023-1981"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://github.com/lathiat/avahi/issues/375"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://bugzilla.redhat.com/show_bug.cgi?id=2185911"
				},
				{
					"source": "security.alpinelinux.org",
					"url": "https://access.redhat.com/security/cve/CVE-2023-1981"
				}
			],
			"published": "2023-05-26T18:15:11Z",
			"modified": "2024-05-22T17:34:18.755386Z"
		}
	],
	"detection": [
		{
			"ecosystem": "alpine:3.19",
			"vulnerable": true,
			"package": {
				"name": "avahi"
			},
			"affected": {
				"type": "apk",
				"range": [
					{
						"lt": "0.8-r14"
					}
				],
				"fixed": [
					"0.8-r14"
				]
			}
		},
		{
			"ecosystem": "alpine:3.20",
			"vulnerable": true,
			"package": {
				"name": "avahi"
			},
			"affected": {
				"type": "apk",
				"range": [
					{
						"lt": "0.8-r14"
					}
				],
				"fixed": [
					"0.8-r14"
				]
			}
		}
	],
	"data_source": "alpine-osv"
}
